example/lib/scan/scan_result_view_model.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:blemulator_example/scan/scan_result.dart';
+import 'package:equatable/equatable.dart';
+
+class ScanResultViewModel extends Equatable {
```

Comment:
`ScannedPeripheral`? `ScanItem`? There are a lot of namespace conflicts after all.